scrapers/boundlife.yml

```diff
@@ -0,0 +1,30 @@
+name: "BoundLife"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 'boundlife.com'
+    scraper: sceneScraper
+    
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performerTitleDate: //div[@class="post-title"]
+    scene:
+      Title:
+        selector: $performerTitleDate/h2
+      Date: 
+        selector: $performerTitleDate/h3/span[3]/text()
+        postProcess:
+          - parseDate: 02 January 2006 
+      Details: 
+        selector: //div[@class="entry"]/p
+      Performers: 
+        Name: 
+          selector: $performerTitleDate/h3/span[2]/text()
+      Studio:
+        Name:
+          fixed: Bound Life 
+      # There are images that can be scraped, but quality is bad.
+      #Image: //img[@border="1"]/@src
```

Comment:
```yaml
      Image: //meta[@property="og:image"]/@content[not(contains(.,"blank.jpg"))]|(//a/img[@border="1"])[1]/@src
```
this selects the main image if its present otherwise the first of the lq ones

Reply:
I only saw updates with the blank images. I'll update.